### PR TITLE
Create files for poetry

### PR DESCRIPTION
**Pull request by serhii73**
This change is [Reviewable](https://reviewable.io/reviews/serhii73/ukrdict/18)

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -16,7 +16,6 @@
   - id: fix-encoding-pragma
   - id: flake8
     args: ["--max-line-length=88"]
-  - id: name-tests-test
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: This doesn't look related + you can also use `args: [--django]` to invert the check

File: .travis.yml [hunk 2]
```diff
@@ -4,5 +4,5 @@ dist: xenial
 python: "3.7"
 cache: pip
 before_install:
-- pip install -r requirements.txt
+- pip install pre-commit
```
Comment (webknjaz) [inline .travis.yml]: At some point, you're just gonna need to start using `tox` as a glue for having the same test env locally and in the CI...

File: tests/test_ukrdict.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (webknjaz) [inline tests/test_ukrdict.py]: This is really only needed if you use Python 2.

File: ukrdict/__init__.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-
 from .ukrdict import find_word
 
 __all__ = ["find_word"]
+__version__ = '0.1.0'
```
Comment (webknjaz) [inline ukrdict/__init__.py]: You can retrieve the installed dist version using `pkg_resources`. With this, you now have two places to track for updating the version.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (serhii73) [reply .pre-commit-config.yaml]: Sorry, it is really not full related, poetry by default create a directory for tests and I simply remove this line from config.
It
https://github.com/serhii73/ukrdict/tree/feature/add_poetry/tests
and it
```
id: name-tests-test
```
is incompatible.

File: tests/test_ukrdict.py (hunk 3 above)
Comment (serhii73) [reply tests/test_ukrdict.py]: poetry automatically create this file.
I remove this line and never be used in the python3 program.
Also, I'll remove such config from pre-commit
https://github.com/serhii73/ukrdict/blob/master/.pre-commit-config.yaml#L16

File: ukrdict/__init__.py (hunk 4 above)
Comment (serhii73) [reply ukrdict/__init__.py]: Need more read about it, because I don't understand why need `two places to track for updating the version`

File: tests/test_ukrdict.py (hunk 3 above)
Comment (serhii73) [reply tests/test_ukrdict.py]: poetry automatically create this file but maybe pre-commit created this line.
Sorry, but `scrapy` create 
```
> $ cat example.py                                                                            
# -*- coding: utf-8 -*-
import scrapy

```
such a line. 
[Scrapy](https://docs.scrapy.org) may be making a mistake?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (webknjaz) [reply .pre-commit-config.yaml]: Nope. It's compatible.

File: tests/test_ukrdict.py (hunk 3 above)
Comment (webknjaz) [reply tests/test_ukrdict.py]: No. Scrapy is probably targeting dual python support which isn't needed for you as you declare py3-only requirement

File: ukrdict/__init__.py (hunk 4 above)
Comment (webknjaz) [reply ukrdict/__init__.py]: Every time you release a new version you need to change (re-hardcode) both `pyproject.toml` (1) and `__init__.py` (1) -> 1+1=2

File: tests/test_ukrdict.py (hunk 3 above)
Comment (serhii73) [reply tests/test_ukrdict.py]: I got it. Thanks @webknjaz

**Comment by serhii73** [comment]
I will be work with this https://github.com/serhii73/ukrdict/pull/17

File: ukrdict/__init__.py (hunk 4 above)
Comment (serhii73) [reply ukrdict/__init__.py]:
```
In [3]: import pkg_resources                                                                   

In [4]: pkg_resources.get_distribution("ukrdict").version                                      
Out[4]: '0.1.0'
```
and `tests/ukrdict_test.py` will be like 

```
import pkg_resources

from ukrdict import __version__


def test_version():
    assert __version__ == pkg_resources.get_distribution("ukrdict").version
```
Right @webknjaz ?

File: .pre-commit-config.yaml [hunk 5]
```diff
@@ -16,6 +16,7 @@
   - id: fix-encoding-pragma
   - id: flake8
     args: ["--max-line-length=88"]
+  - id: name-tests-test
```
Comment (webknjaz) [inline .pre-commit-config.yaml]: `args: ["--django"]` would also work for reversed test file names.

File: ukrdict/__init__.py (hunk 4 above)
Comment (webknjaz) [reply ukrdict/__init__.py]: What are you testing for now? I just don't like the idea of having to change the same thing on multiple places over and over again.
